src/test/java/io/nats/client/JetstreamTests.java

```diff
@@ -35,15 +34,7 @@
 
 public class JetstreamTests {
 
-    private static StreamInfo createMemoryStream(JetStream js, String streamName, String subject)
-            throws TimeoutException, InterruptedException {
-        String[] subjects = new String[1];
-        subjects[0] = subject;
-
-        return createMemoryStream(js, streamName, subjects);
-    }
-
-    private static StreamInfo createMemoryStream(JetStream js, String streamName, String[] subjects)
+    private static StreamInfo createMemoryStream(JetStream js, String streamName, String... subjects)
```

Comment:
This signature will accept one (zero actually) or more strings, so we don't need 2 methods